### PR TITLE
chore: add local deployment instructions

### PR DESCRIPTION
**Pull request by amandakarina**
This PR adds instructions to deploy foundation locally,  without Source Repository and without Cloud Build.

### CODE REVIEW

File: 1-org/README.md [hunk 1]
```diff
@@ -209,14 +209,26 @@ See `0-bootstrap` [README-GitHub.md](../0-bootstrap/README-GitHub.md#deploying-s
...
 1. The next instructions assume that you are at the same level of the `terraform-example-foundation` folder.
-Change into the `1-org` folder, copy the Terraform wrapper script, and ensure it can be executed.
+Create `gcp-org` folder, copy `1-org` content and Terraform wrapper script; ensure it can be executed.
 
    ```bash
-   cd terraform-example-foundation/1-org
-   cp ../build/tf-wrapper.sh .
+   mkdir gcp-org
+   cp -R terraform-example-foundation/1-org/* gcp-org/
+   cp terraform-example-foundation//build/tf-wrapper.sh gcp-org/
```
Comment (apeabody) [inline 1-org/README.md:217]: Is this correct?

```suggestion
   cp terraform-example-foundation/build/tf-wrapper.sh gcp-org/
```

File: 1-org/README.md [hunk 2]
```diff
@@ -209,14 +209,26 @@ See `0-bootstrap` [README-GitHub.md](../0-bootstrap/README-GitHub.md#deploying-s
...
+   cp terraform-example-foundation/.gitignore gcp-org
```
Comment (apeabody) [inline 1-org/README.md:218]: ?

```suggestion
   cp terraform-example-foundation/.gitignore gcp-org/
```

File: 2-environments/README.md [hunk 3]
```diff
@@ -180,12 +179,25 @@ See `0-bootstrap` [README-GitHub.md](../0-bootstrap/README-GitHub.md#deploying-s
...
+1. The next instructions assume that you are at the same level of the `terraform-example-foundation` folder. Create the `gcp-environments` folder, copy the Terraform wrapper script and ensure it can be executed.
 
    ```bash
-   cd terraform-example-foundation/2-environments
-   cp ../build/tf-wrapper.sh .
-   chmod 755 ./tf-wrapper.sh
+   mkdir gcp-environments
+   cp -R terraform-example-foundation/2-environments/* gcp-environments/
+   cp terraform-example-foundation/build/tf-wrapper.sh gcp-environments/
+   cp terraform-example-foundation/.gitignore gcp-environments
```
Comment (apeabody) [inline 2-environments/README.md:188]: ?
```suggestion
   cp terraform-example-foundation/.gitignore gcp-environments/
```

File: 3-networks-dual-svpc/README.md [hunk 4]
```diff
@@ -281,12 +281,26 @@ See `0-bootstrap` [README-GitHub.md](../0-bootstrap/README-GitHub.md#deploying-s
...
+1. The next instructions assume that you are at the same level of the `terraform-example-foundation` folder. Create and change into `gcp-network` folder, copy `3-networks-dual-svpc` content, the Terraform wrapper script and ensure it can be executed. Also, initialize git so you can manage versions locally.
 
    ```bash
-   cd terraform-example-foundation/3-networks-dual-svpc
-   cp ../build/tf-wrapper.sh .
-   chmod 755 ./tf-wrapper.sh
+   mkdir gcp-network
+   cp -R terraform-example-foundation/3-networks-dual-svpc/* gcp-network
+   cp terraform-example-foundation/build/tf-wrapper.sh gcp-network/
+   cp terraform-example-foundation/.gitignore gcp-network
```
Comment (apeabody) [inline 3-networks-dual-svpc/README.md:290]: ?
```suggestion
   cp terraform-example-foundation/.gitignore gcp-network/
```

File: 3-networks-hub-and-spoke/README.md [hunk 5]
```diff
@@ -284,12 +285,25 @@ See `0-bootstrap` [README-GitHub.md](../0-bootstrap/README-GitHub.md#deploying-s
...
+1. The next instructions assume that you are at the same level of the `terraform-example-foundation` folder. Create and change into `gcp-network` folder, copy `3-networks-hub-and-spoke` content, the Terraform wrapper script and ensure it can be executed. Also, initialize git so you can manage versions locally.
 
    ```bash
-   cd terraform-example-foundation/3-networks-hub-and-spoke
-   cp ../build/tf-wrapper.sh .
-   chmod 755 ./tf-wrapper.sh
+   mkdir gcp-network
+   cp -R terraform-example-foundation/3-networks-hub-and-spoke/* gcp-network
+   cp terraform-example-foundation/build/tf-wrapper.sh gcp-network/
+   cp terraform-example-foundation/.gitignore gcp-network
```
Comment (apeabody) [inline 3-networks-hub-and-spoke/README.md:294]: ?
```suggestion
   cp terraform-example-foundation/.gitignore gcp-network/
```

File: 3-networks-hub-and-spoke/README.md [hunk 6]
```diff
@@ -284,12 +285,25 @@ See `0-bootstrap` [README-GitHub.md](../0-bootstrap/README-GitHub.md#deploying-s
...
 
-1. The next instructions assume that you are at the same level of the `terraform-example-foundation` folder. Change into `3-networks-hub-and-spoke` folder, copy the Terraform wrapper script and ensure it can be executed.
+1. The next instructions assume that you are at the same level of the `terraform-example-foundation` folder. Create and change into `gcp-network` folder, copy `3-networks-hub-and-spoke` content, the Terraform wrapper script and ensure it can be executed. Also, initialize git so you can manage versions locally.
 
    ```bash
-   cd terraform-example-foundation/3-networks-hub-and-spoke
-   cp ../build/tf-wrapper.sh .
-   chmod 755 ./tf-wrapper.sh
+   mkdir gcp-network
+   cp -R terraform-example-foundation/3-networks-hub-and-spoke/* gcp-network
```
Comment (apeabody) [inline 3-networks-hub-and-spoke/README.md:292]: ?
```suggestion
   cp -R terraform-example-foundation/3-networks-hub-and-spoke/* gcp-network/
```

File: 4-projects/README.md [hunk 7]
```diff
@@ -237,12 +238,26 @@ See `0-bootstrap` [README-GitHub.md](../0-bootstrap/README-GitHub.md#deploying-s
...
 
-1. The next instructions assume that you are at the same level of the `terraform-example-foundation` folder. Change into `4-projects` folder, copy the Terraform wrapper script and ensure it can be executed.
+1. The next instructions assume that you are at the same level of the `terraform-example-foundation` folder. Create and change into `gcp-projects` folder, copy the code, Terraform wrapper script and ensure it can be executed.
 
    ```bash
-   cd terraform-example-foundation/4-projects
-   cp ../build/tf-wrapper.sh .
-   chmod 755 ./tf-wrapper.sh
+   mkdir gcp-projects
+   cp -R  terraform-example-foundation/4-projects/* gcp-projects
```
Comment (apeabody) [inline 4-projects/README.md:245]: ?
```suggestion
   cp -R  terraform-example-foundation/4-projects/* gcp-projects/
```

File: 4-projects/README.md [hunk 8]
```diff
@@ -237,12 +238,26 @@ See `0-bootstrap` [README-GitHub.md](../0-bootstrap/README-GitHub.md#deploying-s
...
+   cp terraform-example-foundation/build/tf-wrapper.sh gcp-projects/
+   cp terraform-example-foundation/.gitignore gcp-projects
```
Comment (apeabody) [inline 4-projects/README.md:247]: ?
```suggestion
   cp terraform-example-foundation/.gitignore gcp-projects/
```

**Comment by apeabody** [comment]
Doesn't appear the error is related to this change (project number vs ID):

```
Step #8 - "verify-org":     org_test.go:251: 
Step #8 - "verify-org":         	Error Trace:	/workspace/test/integration/org/org_test.go:251
Step #8 - "verify-org":         	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.16.1/pkg/tft/terraform.go:638
Step #8 - "verify-org":         	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.16.1/pkg/tft/terraform.go:670
Step #8 - "verify-org":         	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.16.1/pkg/utils/stages.go:31
Step #8 - "verify-org":         	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.16.1/pkg/tft/terraform.go:670
Step #8 - "verify-org":         	Error:      	Not equal: 
Step #8 - "verify-org":         	            	expected: "bigquery.googleapis.com/projects/200531796710/datasets/ds_c_prj_aggregated_logs_analytics"
Step #8 - "verify-org":         	            	actual  : "bigquery.googleapis.com/projects/kic-c-logging-4dje/datasets/ds_c_prj_aggregated_logs_analytics"
Step #8 - "verify-org":         	            	
Step #8 - "verify-org":         	            	Diff:
Step #8 - "verify-org":         	            	--- Expected
Step #8 - "verify-org":         	            	+++ Actual
Step #8 - "verify-org":         	            	@@ -1 +1 @@
Step #8 - "verify-org":         	            	-bigquery.googleapis.com/projects/200531796710/datasets/ds_c_prj_aggregated_logs_analytics
Step #8 - "verify-org":         	            	+bigquery.googleapis.com/projects/kic-c-logging-4dje/datasets/ds_c_prj_aggregated_logs_analytics
Step #8 - "verify-org":         	Test:       	TestOrg
Step #8 - "verify-org":         	Messages:   	log bucket BigQuery dataset ID should match
```

**Comment by amandakarina** [comment]
> Doesn't appear the error is related to this change (project number vs ID):
> 
> ```
> Step #8 - "verify-org":     org_test.go:251: 
> Step #8 - "verify-org":         	Error Trace:	/workspace/test/integration/org/org_test.go:251
> Step #8 - "verify-org":         	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.16.1/pkg/tft/terraform.go:638
> Step #8 - "verify-org":         	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.16.1/pkg/tft/terraform.go:670
> Step #8 - "verify-org":         	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.16.1/pkg/utils/stages.go:31
> Step #8 - "verify-org":         	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.16.1/pkg/tft/terraform.go:670
> Step #8 - "verify-org":         	Error:      	Not equal: 
> Step #8 - "verify-org":         	            	expected: "bigquery.googleapis.com/projects/200531796710/datasets/ds_c_prj_aggregated_logs_analytics"
> Step #8 - "verify-org":         	            	actual  : "bigquery.googleapis.com/projects/kic-c-logging-4dje/datasets/ds_c_prj_aggregated_logs_analytics"
> Step #8 - "verify-org":         	            	
> Step #8 - "verify-org":         	            	Diff:
> Step #8 - "verify-org":         	            	--- Expected
> Step #8 - "verify-org":         	            	+++ Actual
> Step #8 - "verify-org":         	            	@@ -1 +1 @@
> Step #8 - "verify-org":         	            	-bigquery.googleapis.com/projects/200531796710/datasets/ds_c_prj_aggregated_logs_analytics
> Step #8 - "verify-org":         	            	+bigquery.googleapis.com/projects/kic-c-logging-4dje/datasets/ds_c_prj_aggregated_logs_analytics
> Step #8 - "verify-org":         	Test:       	TestOrg
> Step #8 - "verify-org":         	Messages:   	log bucket BigQuery dataset ID should match
> ```

I think PR #1359 will fix this.

**Pull request merged**